QML_HUD/README.md

```
@@ -7,6 +7,51 @@ theme.json
 
 ## Integrate with your tools
 
+# QML HUD Components
+
+## AbstractControl
```

Markdownlint (0.37.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)